aspnetcore/blazor/components.md

```diff
@@ -889,7 +889,7 @@ Consider the following Pet Details component, which can be manually built into a
 }
 ```
 
-In the following example, the loop in the `CreateComponent` method generates three Pet Details components. When calling `RenderTreeBuilder` methods to create the components (`OpenComponent` and `AddAttribute`), sequence numbers are source code line numbers. The Blazor difference algorithm relies on the sequence numbers corresponding to distinct lines of code, not distinct call invocations. When creating a component with `RenderTreeBuilder` methods, hardcode the arguments for sequence numbers. **Using a calculation or counter to generate the sequence number can lead to poor performance.**
+In the following example, the loop in the `CreateComponent` method generates three Pet Details components. When calling `RenderTreeBuilder` methods to create the components (`OpenComponent` and `AddAttribute`), sequence numbers are source code line numbers. The Blazor difference algorithm relies on the sequence numbers corresponding to distinct lines of code, not distinct call invocations. When creating a component with `RenderTreeBuilder` methods, hardcode the arguments for sequence numbers. **Using a calculation or counter to generate the sequence number can lead to poor performance.** For more information, see [Why sequence numbers should relate to code line numbers, not execution order (SteveSandersonMS/sequence-number.md Gist)](https://gist.github.com/SteveSandersonMS/ec232992c2446ab9a0059dd0fbc5d0c3).
```

Comment:
I'm really against having links to external blog posts/gists/etc as as substitute for documenting our framework features. Can we please just move the relevant parts of Steve's gist into our official docs?

Reply:
Sure ... I'll see if I can condense his content a bit and move it over here.